components/brave_rewards/browser/rewards_service_factory.cc

```diff
@@ -30,7 +30,7 @@
 #endif
 
 namespace {
-#if BUILDFLAG(BRAVE_REWARDS_ENABLED)
+#if BUILDFLAG(BRAVE_REWARDS_ENABLED) && !defined(OS_ANDROID)
```

Comment:
why this change?

Reply:
we don't have guest and tor profiles on Android. There is only one

Reply:
this isn't about tor or guest, it's regular vs incognito so it does apply to android

Reply:
@SergeyZhukovsky for now until I make the fix you do need this code

Reply:
fyi - this is critical @SergeyZhukovsky before merge

Reply:
@bridiver just to make it clear. Do you want to revert this change?

Reply:
Removed